utilities/configure_scripts/configure_dealii.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+## ---------------------------------------------------------------------
+##
+## PartExa - A Particle Library for the Exa-Scale
+##
+## Copyright (C) 2022 by the PartExa authors
+##
+## This program is free software: you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation, either version 3 of the License, or
+## (at your option) any later version.
+##
+## This program is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+## GNU General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with this program. If not, see <https://www.gnu.org/licenses/>.
+##
+## ---------------------------------------------------------------------
+
+# You may want to deactivate this line
+rm -rf CMakeFiles/ CMakeCache.txt
+
+# It is recommended to copy this file to your deal.II build directory via
+# ```bash
+# cd <dealii_build>
+# cp <partexa_source>/utilities/configure_scripts/configure_partexa.sh .
+# ```
```

Comment:
I still prefer to remove the bash instructions here in favor of the Sphinx documentation. Though, we can give a  a short one line statement here, e.g., "Copy this configuration file to new build directory" or something like that.